src/main/resources/resources/logisim/strings/gui/gui_nl.properties

```diff
@@ -681,10 +681,10 @@ aboutDialogCopyDetails = Details kopiëren
 #
 creditsRoleFork = Vork van origineel project
 creditsRoleOriginal = Originele versie
-# ==> creditsBuildInfo =
-# ==> creditsCompiled =
-# ==> creditsDevelopedBy =
-# ==> creditsDevelopedByAndOthers =
+creditsBuildInfo = Build info
+creditsCompiled = Gecompileerd op: %s
+creditsDevelopedBy = Ontwikkeld door
+creditsDevelopedByAndOthers = en anderen...
```

Comment:
Use horizontal ellipsis character "…" instead of three dots.

Reply:
i mean this: Merge branch 'logisim-evolution:main' into develop. is this ok?

Reply:
I think the PR from your `develop` branch is OK: GitHub shows just modifications relevant to this PR. Therefore, merging this PR shouldn't have any bad side effects.